js/main.js

```diff
@@ -1 +1,9 @@
+function addPoem() {
+  let ul = document.getElementById("list");
+  let li = document.createElement("li");
+  let poem = document.getElementById("poem");
+  let poemName = poem;
 
+  li.appendChild(poemName);
```

Comment:
I know that it has something to with my JS here.  I don't want to _appendChild_ but I'm not sure how to push a copy of the text to the "saved content" list